assets/scss/main.scss

```diff
@@ -170,6 +157,29 @@ a {
   }
 }
 
+.link {
+  @include link;
+}
+
+.link-no-decoration {
+  color: inherit;
+  text-decoration: none;
+
+  &:hover, &:active, &:visited {
+    text-decoration: none;
+  }
+}
+
+.hover-list-item {
+  &:hover,
+    &:focus {
+      color: $gray-50;
+      background-color: $gray-10;
+      text-decoration: underline;
```

Comment:
Please remove underline. It looks really bad with that property, adding gray color as you did is sufficient.